### PR TITLE
Added AWS RekognitionService service

### PR DESCRIPTION
**Pull request by robmoorman**
Proof of Concept code for adding AWS Rekognition as a second service (see issue #1 for more details)

screen shot 2016-12-04 at 16 26 06

### CODE REVIEW

**Comment by robmoorman** [comment]
He @marteinn please see the PR for AWS support as well.

Found that the abstraction if pretty simple to use. Maybe we need to do some small design considerations, like exception handling if the image couldn't be downloaded etc. and some good docs.

Do you have time/plans to make this module production ready. I would help with that as I really like suggest this to x of my clients.

File: wagtailaltgenerator/providers/rekognition_service.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        description = None
+        tags = []
+
+        if image_data.status_code == 200:
+            response = self.client.detect_labels(
+                Image={
+                    'Bytes': image_data.content,
+                },
+                MinConfidence=50
```
Comment (robmoorman) [inline wagtailaltgenerator/providers/rekognition_service.py:27]: Minimum of 50% confidence. Found that it eassier to just remove some tags which you wouldn't want to be added is prettier for usage than adding tags manually. Maybe we need to make this percentage editable via settings though...

File: wagtailaltgenerator/providers/rekognition_service.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    def __init__(self, *args, **kwargs):
+        self.client = boto3.client('rekognition')
+        super(RekognitionService, self).__init__(*args, **kwargs)
+
+    def describe(self, image):
+        image_url = image.file.url
+        image_data = requests.get(image_url)
+
+        description = None
```
Comment (robmoorman) [inline wagtailaltgenerator/providers/rekognition_service.py:19]: Description isn't returned by AWS

**Review by marteinn — APPROVED** [review]
Great work! I'm merging this into the branch and continuing with some fine tuning.

File: wagtailaltgenerator/providers/rekognition_service.py (hunk 1 above)
Comment (marteinn) [reply wagtailaltgenerator/providers/rekognition_service.py:27]: Good idea! I will check if it's possible to translate this setting over to the cognitive provider.

**Pull request merged**